pkg/ebpf/bpf_client.go

```diff
@@ -856,10 +857,7 @@ func mergeDuplicateL4Info(ports []v1alpha1.Port) []v1alpha1.Port {
 func (l *bpfClient) computeMapEntriesFromEndpointRules(firewallRules []EbpfFirewallRules) (map[string][]byte, error) {
```

Comment:
As an enhancement, could you document the behavior of this function as function doc string itself. We pack a lot inside the function here.